index.js

```diff
@@ -1,26 +1,32 @@
 import axios from "axios";
 import dotenv from "dotenv";
+import fs from "fs";
 dotenv.config();
 
 const riotApi = process.env.API_URL;
 const baseUrl = "https://eu.api.riotgames.com/val/content/v1/contents"
 
+let datas;
 const getData = async () => {
     const url = `${baseUrl}?api_key=${riotApi}`;
-
     try {
         const response = await axios.get(url)
```

Comment:
const response = await axios.get("apilink");

    if (res.status === 429){
      ...code
    }
    if (response.status === 200) {
      console.log(response.data);
    }

Try doing something like this, as code will be much cleaner with async/await than using '.then'